### ISSUE TITLE
Feature Request: Add phone number to successfully verified PIN check response

### ISSUE THREAD

**Issue by sirolad** [Vonage/vonage-node-sdk#254]
When I call the API via `Nexmo.verify.check` , The post data for the request contains just
{request_id, PIN} and returns {
  request_id: 'XXXXXXXX',
  status: '0',
  event_id: 'XXXXXXXX',
  price: '0.10000000',
  currency: 'EUR'
}
It would be helpful to have this contain the verified phone number as well.

**Comment by AlexLakatos** [comment]
We're passing the API response directly, it's not a nexmo-node feature. 

Thanks for filing the issue btw, I'll forward this to our Product team as a feature request for the API! It would help them prioritise this if you explained your use case for having the phone number as well in that response.

**Comment by sirolad** [comment]
Thanks @AlexLakatos , my use case is when verifying I have to pass the phone number along the PIN and request id to the server so that I can tag the verified user to a phone number I already know. though if I had saved the request Id in the db, I could have use that to reference the user but because it is shortlived(5 mins) I did not see any point doing so. Having the phone number with the verified response can save me the stress, I will only need to check the number against what I have on the server. Thanks
